INFO - main.py - 2025-06-26 11:24:48,083 - Using device: cuda:7
INFO - main.py - 2025-06-26 11:24:48,083 - Writing log file to ./results/card_onehot_conditional_results/10steps/nn/run_1/f_phi_prior_cat_f_phi/f_phi_supervised/logs/firefox_fiber_2d/split_0
INFO - main.py - 2025-06-26 11:24:48,083 - Exp instance id = 1346321
INFO - main.py - 2025-06-26 11:24:48,083 - Exp comment = 
ERROR - main.py - 2025-06-26 11:24:51,101 - Traceback (most recent call last):
  File "/home/kwakrhkr59/XAI_WF/diffusion/classification/main.py", line 301, in main
    y_majority_vote_accuracy_all_steps_list = runner.test_image_task()
  File "/home/kwakrhkr59/XAI_WF/diffusion/classification/card_classification.py", line 855, in test_image_task
    states = torch.load(os.path.join(log_path, "ckpt.pth"),
  File "/home/kwakrhkr59/anaconda3/envs/diff/lib/python3.10/site-packages/torch/serialization.py", line 699, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/kwakrhkr59/anaconda3/envs/diff/lib/python3.10/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/kwakrhkr59/anaconda3/envs/diff/lib/python3.10/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './results/card_onehot_conditional_results/10steps/nn/run_1/f_phi_prior_cat_f_phi/f_phi_supervised/logs/firefox_fiber_2d/split_0/ckpt.pth'

INFO - main.py - 2025-06-26 11:25:15,527 - Using device: cuda:7
INFO - main.py - 2025-06-26 11:25:15,527 - Writing log file to ./results/card_onehot_conditional_results/10steps/nn/run_1/f_phi_prior_cat_f_phi/f_phi_supervised/logs/firefox_fiber_2d/split_0
INFO - main.py - 2025-06-26 11:25:15,527 - Exp instance id = 1346652
INFO - main.py - 2025-06-26 11:25:15,527 - Exp comment = 
INFO - card_classification.py - 2025-06-26 11:25:18,578 - Loading from: ./results/card_onehot_conditional_results/10steps/nn/run_1/f_phi_prior_cat_f_phi/f_phi_supervised/logs/firefox_fiber_2d/split_0/ckpt_last.pth
ERROR - main.py - 2025-06-26 11:25:18,590 - Traceback (most recent call last):
  File "/home/kwakrhkr59/XAI_WF/diffusion/classification/main.py", line 301, in main
    y_majority_vote_accuracy_all_steps_list = runner.test_image_task()
  File "/home/kwakrhkr59/XAI_WF/diffusion/classification/card_classification.py", line 863, in test_image_task
    model.load_state_dict(states[0], strict=True)
  File "/home/kwakrhkr59/anaconda3/envs/diff/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1604, in load_state_dict
    raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
RuntimeError: Error(s) in loading state_dict for ConditionalModel:
	Missing key(s) in state_dict: "encoder_x.f.0.weight", "encoder_x.f.1.weight", "encoder_x.f.1.bias", "encoder_x.f.1.running_mean", "encoder_x.f.1.running_var", "encoder_x.f.4.0.conv1.weight", "encoder_x.f.4.0.bn1.weight", "encoder_x.f.4.0.bn1.bias", "encoder_x.f.4.0.bn1.running_mean", "encoder_x.f.4.0.bn1.running_var", "encoder_x.f.4.0.conv2.weight", "encoder_x.f.4.0.bn2.weight", "encoder_x.f.4.0.bn2.bias", "encoder_x.f.4.0.bn2.running_mean", "encoder_x.f.4.0.bn2.running_var", "encoder_x.f.4.1.conv1.weight", "encoder_x.f.4.1.bn1.weight", "encoder_x.f.4.1.bn1.bias", "encoder_x.f.4.1.bn1.running_mean", "encoder_x.f.4.1.bn1.running_var", "encoder_x.f.4.1.conv2.weight", "encoder_x.f.4.1.bn2.weight", "encoder_x.f.4.1.bn2.bias", "encoder_x.f.4.1.bn2.running_mean", "encoder_x.f.4.1.bn2.running_var", "encoder_x.f.5.0.conv1.weight", "encoder_x.f.5.0.bn1.weight", "encoder_x.f.5.0.bn1.bias", "encoder_x.f.5.0.bn1.running_mean", "encoder_x.f.5.0.bn1.running_var", "encoder_x.f.5.0.conv2.weight", "encoder_x.f.5.0.bn2.weight", "encoder_x.f.5.0.bn2.bias", "encoder_x.f.5.0.bn2.running_mean", "encoder_x.f.5.0.bn2.running_var", "encoder_x.f.5.0.downsample.0.weight", "encoder_x.f.5.0.downsample.1.weight", "encoder_x.f.5.0.downsample.1.bias", "encoder_x.f.5.0.downsample.1.running_mean", "encoder_x.f.5.0.downsample.1.running_var", "encoder_x.f.5.1.conv1.weight", "encoder_x.f.5.1.bn1.weight", "encoder_x.f.5.1.bn1.bias", "encoder_x.f.5.1.bn1.running_mean", "encoder_x.f.5.1.bn1.running_var", "encoder_x.f.5.1.conv2.weight", "encoder_x.f.5.1.bn2.weight", "encoder_x.f.5.1.bn2.bias", "encoder_x.f.5.1.bn2.running_mean", "encoder_x.f.5.1.bn2.running_var", "encoder_x.f.6.0.conv1.weight", "encoder_x.f.6.0.bn1.weight", "encoder_x.f.6.0.bn1.bias", "encoder_x.f.6.0.bn1.running_mean", "encoder_x.f.6.0.bn1.running_var", "encoder_x.f.6.0.conv2.weight", "encoder_x.f.6.0.bn2.weight", "encoder_x.f.6.0.bn2.bias", "encoder_x.f.6.0.bn2.running_mean", "encoder_x.f.6.0.bn2.running_var", "encoder_x.f.6.0.downsample.0.weight", "encoder_x.f.6.0.downsample.1.weight", "encoder_x.f.6.0.downsample.1.bias", "encoder_x.f.6.0.downsample.1.running_mean", "encoder_x.f.6.0.downsample.1.running_var", "encoder_x.f.6.1.conv1.weight", "encoder_x.f.6.1.bn1.weight", "encoder_x.f.6.1.bn1.bias", "encoder_x.f.6.1.bn1.running_mean", "encoder_x.f.6.1.bn1.running_var", "encoder_x.f.6.1.conv2.weight", "encoder_x.f.6.1.bn2.weight", "encoder_x.f.6.1.bn2.bias", "encoder_x.f.6.1.bn2.running_mean", "encoder_x.f.6.1.bn2.running_var", "encoder_x.f.7.0.conv1.weight", "encoder_x.f.7.0.bn1.weight", "encoder_x.f.7.0.bn1.bias", "encoder_x.f.7.0.bn1.running_mean", "encoder_x.f.7.0.bn1.running_var", "encoder_x.f.7.0.conv2.weight", "encoder_x.f.7.0.bn2.weight", "encoder_x.f.7.0.bn2.bias", "encoder_x.f.7.0.bn2.running_mean", "encoder_x.f.7.0.bn2.running_var", "encoder_x.f.7.0.downsample.0.weight", "encoder_x.f.7.0.downsample.1.weight", "encoder_x.f.7.0.downsample.1.bias", "encoder_x.f.7.0.downsample.1.running_mean", "encoder_x.f.7.0.downsample.1.running_var", "encoder_x.f.7.1.conv1.weight", "encoder_x.f.7.1.bn1.weight", "encoder_x.f.7.1.bn1.bias", "encoder_x.f.7.1.bn1.running_mean", "encoder_x.f.7.1.bn1.running_var", "encoder_x.f.7.1.conv2.weight", "encoder_x.f.7.1.bn2.weight", "encoder_x.f.7.1.bn2.bias", "encoder_x.f.7.1.bn2.running_mean", "encoder_x.f.7.1.bn2.running_var", "encoder_x.g.weight", "encoder_x.g.bias". 
	Unexpected key(s) in state_dict: "encoder_x.0.weight", "encoder_x.0.bias", "encoder_x.1.weight", "encoder_x.1.bias", "encoder_x.1.running_mean", "encoder_x.1.running_var", "encoder_x.1.num_batches_tracked", "encoder_x.3.weight", "encoder_x.3.bias", "encoder_x.4.weight", "encoder_x.4.bias", "encoder_x.4.running_mean", "encoder_x.4.running_var", "encoder_x.4.num_batches_tracked", "encoder_x.6.weight", "encoder_x.6.bias". 
	size mismatch for lin1.lin.weight: copying a param with shape torch.Size([128, 20]) from checkpoint, the shape in current model is torch.Size([128, 150]).
	size mismatch for lin4.weight: copying a param with shape torch.Size([10, 128]) from checkpoint, the shape in current model is torch.Size([75, 128]).
	size mismatch for lin4.bias: copying a param with shape torch.Size([10]) from checkpoint, the shape in current model is torch.Size([75]).

